overlay.d/05rhcos/usr/lib/systemd/system-generators/bootloader-migrate-generator

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
What prevents this generator from generating this service on EVERY boot? Is this something we only want to run once?

Reply:
Added some info, let me know if you have more comment, thank you!